.github/workflows/test.yml

```
@@ -25,3 +25,6 @@ jobs:
         with:
           version: ${{ matrix.emacs-version }}
       - uses: leotaku/elisp-check@6a1fc6b11ae66dda13859f8b3474a36dfe3ef225
```

[suggest] To line the existing file, it seems required to have two jobs for `leotaku/elisp-check`. For example:

```suggestion
      - uses: leotaku/elisp-check@6a1fc6b11ae66dda13859f8b3474a36dfe3ef225
        with:
          file: rbs-mode.el
      - uses: leotaku/elisp-check@6a1fc6b11ae66dda13859f8b3474a36dfe3ef225
```